sdk/tests/deqp/functional/gles3/es3fTransformFeedbackTests.js

```diff
@@ -895,9 +895,13 @@ goog.scope(function() {
 
             if (linkFail) {
                 if (!es3fTransformFeedbackTests.isProgramSupported(this.m_progSpec, this.m_bufferMode)) {
-                    throw new Error('Not Supported. Implementation limits exceeded.');
+                    var msg = 'Not Supported. Implementation limits exceeded.';
+                    checkMessage(false, msg);
+                    throw new TestFailedException(msg);
```

Comment:
No, it doesn't report a test failure. I admit that TestFailedException name can be confusing, though.
When a test fails, we usually report test failure to the harness and then throw TestFailedException. This causes the test runner to move on to the next test.
In this case, we're printing a warning message and terminating the test.
All exceptions other than TestFailedException are caught by the test runner and result in test failure.